ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_context.py

```diff
@@ -58,17 +59,22 @@ def create_timer(self, fire_at: Union[datetime, timedelta]) -> task.Task:
         return self.__obj.create_timer(fire_at)
 
     def call_activity(self, activity: Callable[[WorkflowActivityContext, TInput], TOutput], *,
-                      input: TInput = None) -> task.Task[TOutput]:
+                      input: TInput = None,
+                      retry_policy: Optional[RetryPolicy] = None) -> task.Task[TOutput]:
         self._logger.debug(f'{self.instance_id}: Creating activity {activity.__name__}')
         if hasattr(activity, '_dapr_alternate_name'):
+            if retry_policy is None:
+                return self.__obj.call_activity(activity=activity.__dict__['_dapr_alternate_name'],
+                                                input=input)
             return self.__obj.call_activity(activity=activity.__dict__['_dapr_alternate_name'],
-                                            input=input)
+                                            input=input, retry_policy=retry_policy.obj)
         # this return should ideally never execute
         return self.__obj.call_activity(activity=activity.__name__, input=input)
```

Comment:
Although the comment here says that it should never execute ideally, still should we add the retry policy here as well? The retry policy check can be brought outside the `hasattr` condition, similar to this: https://github.com/deepanshua/python-sdk/blob/b2e3bc71853af8d536c743210f277cf77a5f2611/ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_context.py#L90-L91

Reply:
Done.